p2p/pre2p/stdnetwork/network.go

```diff
@@ -66,3 +67,8 @@ func (n *network) AddPeerToAddrBook(peer *typesPre2P.NetworkPeer) error {
 func (n *network) RemovePeerToAddrBook(peer *typesPre2P.NetworkPeer) error {
 	panic("RemovePeerToAddrBook not implemented")
 }
+
+func (n *network) GetBus() modules.Bus  { return nil }
```

Comment:
I understand that you made this a module so that you can get the block height via the bus.

Needs aside, from an overall design standpoint, do you think it makes sense for this to be a module that is started and stopped? Can you think of other approaches?

Reply:
Good point. Maybe we can split the `Module` interface into `Runnable` and `Accessible` such that:

1. Runnable is:
```go
type Runnable interface {
    Start()
    Stop()
}
```

and 2. Accessible is:
```go
type Accessible interface {
    GetBus()
    SetBus()
}
```

such that the `Network` in this case can do without with the runnable functionality but still access (_and be accessed by, if necessary?_) other modules.